### PR TITLE
Add intro to content page and fix ToC bug

### PR DESCRIPTION
**Pull request by peterdavidhamilton**
## Ticket and context

Ticket: [HFEYP-353](https://dfedigital.atlassian.net/browse/ER-353)
Screenshot 2022-08-12 at 10 55 54

- add intro markdown field to content_page and render after the title
- only render TOC if not blank

### Code quality checks
- [ ] All commit messages are meaningful and true

### How can someone see it in review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Follow the steps from the ticket.

### CODE REVIEW

File: db/seeds/content_pages/communication_and_language.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@ interactions:
   parent: communication-and-language
   position: 1
   is_published: true
+  intro: |
```
Comment (ebrett) [inline db/seeds/content_pages/communication_and_language.yml:37]: Just noting that the seed data here is only used when creating a new environment - this won't add the intro in the production environment unless we add it to the content page form.

Comment (peterdavidhamilton) [reply db/seeds/content_pages/communication_and_language.yml:37]: Understood, I thought it wise when bootstrapping. As a new comer I had presumed the ToC would be pre-populated . Would it be wise to add that to the seeds as well?

**Comment by peterdavidhamilton** [comment]
@ebrett middleware is causing a single failure. Can you assist in unpicking?

File: db/seeds/content_pages/communication_and_language.yml (hunk 1 above)
Comment (ebrett) [reply db/seeds/content_pages/communication_and_language.yml:37]: The ToC came later (as an afterthought really) - I don't know what pages have it, but I think the presumption was most pages would not.  I'd say leave it.

**Comment by peterdavidhamilton** [comment]
@ebrett middleware failure didn't occur on subsequent run

**Review by agoldstone93 — APPROVED** [review]

**Comment by peterdavidhamilton** [comment]
Auto PR comment didn't work, however this is the seeded page with an intro (no ToC is seeded though).

https://eyfs-review-pr-611.london.cloudapps.digital/communication-and-language/interactions

File: app/views/admin/content_pages/_form.html.erb [hunk 2]
```diff
@@ -43,6 +45,9 @@
 
     <div class="govuk-grid-column-one-half">
         <p class="govuk-heading-m">Preview</p>
+        <div id='content-list-render' class='gem-c-govspeak'>
+          <%= translate_markdown(content_page.intro) %>
+        </div>
         <div id='content-list-render' class='gem-c-govspeak'>
           <%= translate_markdown(content_page.content_list) %>
         </div>
```
Comment (ebrett) [inline app/views/admin/content_pages/_form.html.erb:53]: Should you have two identical `id` elements? maybe `page-intro-render`

**Review by ebrett — CHANGES_REQUESTED** [review]
Fix the id, but otherwise looks good

File: app/views/admin/content_pages/_form.html.erb (hunk 2 above)
Comment (peterdavidhamilton) [reply app/views/admin/content_pages/_form.html.erb:53]: Oh boy thank you, I caught the other one but just removed the id completely.

**Review by ebrett — APPROVED** [review]

**Pull request merged**